src/Cli/dotnet/Commands/Test/TestingPlatformCommand.cs

```diff
@@ -29,50 +29,63 @@ public int Run(ParseResult parseResult)
         int exitCode = ExitCode.Success;
         try
         {
-            ValidationUtility.ValidateMutuallyExclusiveOptions(parseResult);
+            exitCode = RunInternal(parseResult);
```

Comment:
Moved the content to separate method, so we have easier time getting he final exit code and returning it.